protos/mocap/mocap.proto

```diff
@@ -0,0 +1,164 @@
+
+/**
+ * Motion Capture allow vehicles to navigate when a global
+ * position source is unavailable or unreliable
+ * (e.g. indoors, or when flying under a bridge. etc.).
+ */
+service MocapService {
+    // Send Global position/attitude estimate from a vision source.
+    rpc SetVisionPositionEstimate(SetVisionPositionEstimateRequest) returns(SetVisionPositionEstimateResponse) {}
+    // Send motion capture attitude and position.
+    rpc SetAttitudePositionMocap(SetAttitudePositionMocapRequest) returns(SetAttitudePositionMocapResponse) {}
+    // Send odometry information with an external interface.
+    rpc SetOdometry(SetOdometryRequest) returns(SetOdometryResponse) {}
+}
+
+message SetVisionPositionEstimateRequest {
+    VisionPositionEstimate vision_position_estimate = 1;
+}
+message SetVisionPositionEstimateResponse {
+    MocapResult mocap_result = 1;
+}
+
+message SetAttitudePositionMocapRequest {
+    AttitudePositionMocap attitude_position_mocap = 1;
+}
+message SetAttitudePositionMocapResponse {
+    MocapResult mocap_result = 1;
+}
+
+message SetOdometryRequest {
+    Odometry odometry = 1;
+}
+message SetOdometryResponse {
+    MocapResult mocap_result = 1;
+}
+
+// Global position/attitude estimate from a vision source.
+message VisionPositionEstimate {
+    uint64 time_usec = 1; // PositionBody frame timestamp UNIX Epoch time (0 to use Backend timestamp)
+    PositionBody position_body = 2; // Global position (m)
+    AngleBody angle_body = 3; // Body angle (rad).
+    Covariance pose_covariance = 4; // Pose cross-covariance matrix.
+}
+
+// Motion capture attitude and position
+message AttitudePositionMocap {
+    uint64 time_usec = 1; // PositionBody frame timestamp UNIX Epoch time (0 to use Backend timestamp)
+    Quaternion q = 2; // Attitude quaternion (w, x, y, z order, zero-rotation is 1, 0, 0, 0)
+    PositionBody position_body = 3; // Body Position (NED)
+    Covariance pose_covariance = 4; // Pose cross-covariance matrix.
+}
+
+// Odometry message to communicate odometry information with an external interface.
+message Odometry {
+    // Mavlink frame id
+    enum MavFrame {
+        STUB = 0; // Stub for Protobuf
+        BODY_FRD = 12; // Body fixed frame of reference, Z-down (x: forward, y: right, z: down). Replacement for MAV_FRAME_BODY_NED, MAV_FRAME_BODY_OFFSET_NED.
+        BODY_FLU = 13; // Body fixed frame of reference, Z-up (x: forward, y: left, z: up).
+        MOCAP_NED = 14; // Odometry local coordinate frame of data given by a motion capture system, Z-down (x: north, y: east, z: down).
+        VISION_NED = 16; // Odometry local coordinate frame of data given by a motion capture system, Z-up (x: east, y: north, z: up).
+        LOCAL_FRD = 20; // Forward, Right, Down coordinate frame. This is a local frame with Z-down and arbitrary F/R alignment (i.e. not aligned with NED/earth frame). Replacement for MAV_FRAME_MOCAP_NED, MAV_FRAME_VISION_NED, MAV_FRAME_ESTIM_NED.
```

Comment:
Will be in use
https://mavlink.io/en/messages/common.html#MAV_FRAME_LOCAL_FRD

Reply:
UPD
Already in use
https://github.com/PX4/Firmware/blob/master/src/modules/mavlink/mavlink_receiver.cpp#L1276